scripts/rpcConfig/constants/finality.ts

```diff
@@ -0,0 +1,652 @@
+import { ChainSlug } from "../../../src/enums/chainSlug";
+
+import {
+  FinalityBucket,
+  FinalityType,
+  ChainFinalityInfo,
+} from "../../../src/socket-types";
+
+export const finality: {
+  [chainSlug in ChainSlug]?: ChainFinalityInfo;
+} = {
+  [ChainSlug.AEVO]: {
```

Comment:
the mapping is too big and not maintainable. lets use defaults based on chain types.
i.e. opstack chain, orbit chain have same defaults that can be overridden if needed.